soh/src/overlays/actors/ovl_En_Sth/z_en_sth.c

```diff
@@ -277,7 +279,11 @@ void EnSth_GivePlayerItem(EnSth* this, GlobalContext* globalCtx) {
         }
     }
 
-    func_8002F434(&this->actor, globalCtx, getItemId, 10000.0f, 50.0f);
+    if (!gSaveContext.n64ddFlag) {
```

Comment:
Logical Error: Can it get to this point without setting getItemEntry?

Reply:
Same suggestion as for fishing.